hail/python/hail/docs/change_log.md

```diff
@@ -23,6 +23,17 @@ relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
 
+---
+
+## Version 0.2.91
+
+Release 2022-03-18
+
+### Bug fixes
```

Comment:
https://github.com/hail-is/hail/pull/11563 was also a bug fix

Reply:
And depending on how desperately the Leo people need a release, I'd like to get #11611 in this as well